File: DartMobileClient/lib/model/tramway_line.dart
part of model;

class TramwayLine extends TransportationLine {

  TramwayLine();

  TramwayLine.fromMap(Map tramwayLineMap) : super.fromMap(tramwayLineMap);

  Map toJson() {
    Map json = super.toJson();
    json["@type"] = "TramwayLine";
    return json;
  }
  
}

File: Transportation-model/src/main/java/com/transportation/transportation/model/entities/TramwayLine.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package com.transportation.transportation.model.entities;

/**
 *
 * @author youssef
 */
public class TramwayLine extends TransportationLine {

    public TramwayLine() {
    }

    public TramwayLine(TransportationLine transportationLine) {
        super(transportationLine);
    }

    @Override
    public TransportationLine clone() {
        return new TramwayLine(this);
    }
    
}
